core-bundle/src/Resources/config/services.yml

```diff
@@ -99,7 +99,6 @@ services:
     # Backwards compatibility
     contao.controller.backend_csv_import:
         alias: Contao\CoreBundle\Controller\BackendCsvImportController
-        deprecated: Using the "%alias_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "Contao\CoreBundle\Controller\BackendCsvImportController" instead.
```

Comment:
If we want to keep this and also avoid having a `getDeprecationArgs()` method in the `Configuration` class (see above), we would have to require `symfony/dependency-injection` in version 5 only. Not sure if that is worth it, given that we are only talking about a few service alias deprecations. @contao/developers WDYT?

Reply:
I think removing it is not the way to go. We should leave it there and comment it out so people understand why it's commented out. We can bring it back once we're 5.x only.

Reply:
Re-added in 13d8b5b885b4749b1c18174cb566c8d889a2a099.